### DISCUSSION TITLE
Ikea E1812 binding issue with Group 0

### DISCUSSION THREAD

**Discussion by balthus** [Koenkk/zigbee2mqtt#14219] [Devices]
Hello,
I prefer to start with a discussion than with a bug.
I tried for many hours to bind an ikea remote E1812 to a group that contains an ikea bulb([LED2003G10).

I first bind the 2 devices to z2m, then I created a group, added the lamp in it. Then I go to the remote settings and add a bind to the group:
![image](https://user-images.githubusercontent.com/16748341/192591123-4e9dc288-aee3-4d49-9f40-dff2de7736c8.png)

Everything seems fine, but when I click on the remote, the bulb doesn't turn on.
I can turn on the bulb manually from the device exposed values.

I also check the debug and the remote is still always associated to group 0:
![image](https://user-images.githubusercontent.com/16748341/192591604-3288bad4-d482-4dc8-8ff9-09dbdab12fe2.png)

I tried many times to delete group, unbind, unassociate/reassociate but nothing works.

If anyone has some idea or some help. I tried to read post/bugs but it didn't solve my issue.

I also planned to upgrade the coordinator firmware (currently  zStack3x0 20210120 with C1352P2_CC2652P_launchpad_coordinator_20220219.hex) but I'm a bit scared to break the whole zigbee network.
Can I reflash it using the documented procedure? Should we erase everything before overriding the firmware? I don't know if there are some info stored on the coordinator...

Thanks for any help ;)

**Comment by balthus** [comment]
bump, no one has some ideas?

**Reply by sjorge** [reply]
Is the firmware on the remote up to date? If so you can no longer bind it to a group but have to bind to the device directly.

**Comment by balthus** [comment]
Yes, I think I did update it via OTA.
Right now it's firmware 
20210727 | 2.3.080

I will check if it's the latest. Thanks
